src/components/product-page/BreadCrumbs.vue

```diff
@@ -0,0 +1,28 @@
+<script setup lang="ts">
```

Comment:
In theory we want to reuse it on other pages as well. Would be nice to move it to the components folder and make it work without parameters. Especially that you can get the productName from useProduct

Reply:
parameters or props?